build.gradle

```diff
@@ -265,62 +265,47 @@ dependencyManagement {
     // CVE-2021-29425
     dependency group: 'commons-io', name: 'commons-io', version: '2.11.0'
 
-    dependencySet(group: 'org.springframework.cloud', version: '3.1.5') {
+    dependencySet(group: 'org.springframework.cloud', version: '4.0.2') {
       entry 'spring-cloud-starter-openfeign'
       entry 'spring-cloud-openfeign-core'
     }
-
-    dependencySet(group: 'org.springframework.security', version: '6.0.3') {
-      entry 'spring-security-core'
-      entry 'spring-security-test'
-    }
   }
   imports {
-    mavenBom 'org.springframework.cloud:spring-cloud-dependencies:2021.0.3'
+    mavenBom 'org.springframework.cloud:spring-cloud-dependencies:2022.0.2'
```

Comment:
This has been removed in LAU

Reply:
Hi Martin , yes it is required and it's there in LAU as well.